blazor/chart/how-to/synchronized-chart.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
Have to prepare content as how to section step by step guidelines. tooltip, crosshair, and zooming.

Reply:
resolved